run-pr-tests

```diff
@@ -151,11 +153,48 @@ fi
 git log --oneline --merges ${CMSSW_VERSION}..
 git cms-checkdeps -A -a
 
-if [ "X$AUTO_POST_MESSAGE" = Xtrue ]; then
-  $WORKSPACE/cms-bot/report-pull-request-results TESTS_RUNNING --pr $PULL_REQUEST_NUMBER -c $LAST_COMMIT --pr-job-id $PULL_REQUEST_JOB_ID --add-message "Compiling"
+$WORKSPACE/cms-bot/report-pull-request-results TESTS_RUNNING --pr $PULL_REQUEST_NUMBER -c $LAST_COMMIT --pr-job-id $PULL_REQUEST_JOB_ID --add-message "Compiling" $DRY_RUN
+
+# #############################################
+# test compilation with Clang
+# ############################################
+echo 'test clang compilation'
+
+#CONFIG_MAP_URL="https://raw.githubusercontent.com/cms-sw/cms-bot/master/config.map"
+CONFIG_MAP_URL='https://github.com/nclopezo/cms-bot/blob/test-compilation-clang-2b/config.map'
+
+NEED_CLANG_TEST=false
+if curl $CONFIG_MAP_URL | grep $RELEASE_QUEUE | grep PRS_TEST_CLANG= | grep SCRAM_ARCH=$ARCHITECTURE; then
+  NEED_CLANG_TEST=true
+fi
+
+if [ "X$TEST_CLANG_COMPILATION" = Xtrue -a $NEED_CLANG_TEST = true ]; then
+  $WORKSPACE/cms-bot/report-pull-request-results TESTS_RUNNING --pr $PULL_REQUEST_NUMBER -c $LAST_COMMIT --pr-job-id $PULL_REQUEST_JOB_ID --add-message "Testing Clang compilation" $DRY_RUN
+
+  #first, add the command to the log
+  echo "scram b vclean && scram build -k -j $(getconf _NPROCESSORS_ONLN) USER_CXXFLAGS='-fsyntax-only' COMPILER='llvm compile'" > $WORKSPACE/buildClang.log
+
+  scram b vclean && scram build -k -j $(getconf _NPROCESSORS_ONLN) USER_CXXFLAGS='-fsyntax-only' COMPILER='llvm compile' 2>&1 | tee -a $WORKSPACE/buildClang.log
```

Comment:
ah I see that this is the most recent diff. Here it does scram b vclean. would it be ok like this? or should I put it back at the end?